doc/ApiDocGeneration/Generate-DocIndex.ps1

```diff
@@ -60,7 +60,7 @@ $ServiceMapping = @{
     "storage"="Storage"; "storagecache"="Storage Cache";
     "storagesync"="Storage Sync"; "storsimple"="Stor Simple";
     "storsimple8000series"="Stor Simple 8000 series"; "streamanalytics"="Stream Analytics";
-    "subscription"="Subscription";
+    "subscription"="Subscription"; "textanalytics"="Text Analytics";
```

Comment:
Thanks for adding this @chidozieononiwu . Could you explain to me what this is? When do we have to modify this file?

Reply:
This is just for the Github.io index. So that it shows up as a friendly name `Text Analytics` instead of `textanalytics`

Reply:
Oh cool. do we need to do this every time we publish a new SDK?
Could we document this step in the release doc or somewhere appropriate so next time we know we need to update this file?

Reply:
Only when a new SDK in on-boarded into the repo. I will add a step in CONTRIBUTING.md.

Reply:
and maybe here:? https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md